### ISSUE TITLE
FR: Make users read-only

### ISSUE THREAD

**Issue by totti4ever** [eikek/docspell#378]
I don't trust my wife more than I need the documents stored in Docspell :-)

maybe, one could mark folders to be read-only besides owner, more complex would be to set the right per user and folder

**Comment by eikek** [comment]
haha :-) Yeah, I really see your point. I admit that I refrain a little from adding access control things, because it adds quite some complexity and I fear a little going down this path would increase dev effort a lot. In the beginning I decided to not add access control within a collective to keep things a bit easier (to use and maintain). The folder with its owner is just meant as a "visibility feature", in contrast to a "security feature", but I know I opened something with that ;). It would also add some burden to the mass-edit-feature (#253). What kind of protection do you have in mind? Only deletion of attachments (and item) or also writing all metadata? I'd think protecting from deletion might be enough?

I think I could maybe be convinced to add a "protected folder" feature, where only the owner has write access. I don't think I want to add fine grained access control like right per user.

**Comment by totti4ever** [comment]
Indeed, I totally agree with keeping the complexity low...it is still one collective! I mean, basically through backup one would always have the chance to get the docs back...but you have to know that it's gone :-D

I think, such a protected folder is the easiest and still helpful solution. I also would only restrict deletion and maybe file renaming (although I don't care too much until the Samba module arrives)

**Comment by eikek** [comment]
Ok, that's good! Then I would start with adding something like a protected flag to a folder and for a first shot deny deleting any file/item.

**Comment by jza34** [comment]
I am also interested in making a folder read only for users, but not for the owner of course!
Use case: give to users an access to documents in folders but with no possibility to change anything, like removing documents from this folder and make them accessible to anybody!

Are you still looking for a way to mark a folder protected?

Your software give me hope as I am looking for an EDMS open for read only to external users. I try Alfresco, Mayan, Paperless-ngx, etc. and each of them got pros and cons...

Please let me know about this protected folder feature still in the roadmap or not.

Thanks for your great opensource software anyway, very useful in many use cases!
